lang/cs/Org.Apache.REEF.IO.TestClient/HadoopFileInputPartitionTest.cs

```diff
@@ -45,6 +46,7 @@ internal static bool TestWithByteDeserializer()
             var serializerConf = TangFactory.GetTang().NewConfigurationBuilder()
                 .BindImplementation<IFileDeSerializer<IEnumerable<byte>>, ByteSerializer>(GenericType<IFileDeSerializer<IEnumerable<byte>>>.Class,
                     GenericType<ByteSerializer>.Class)
+                .BindNamedParam<CopyToLocal, bool>("true")
```

Comment:
In the current Tang API, the value of a named parameter for preliminary type like int, bool, are all use string format.